### PR TITLE
E2E 🟢 

### PR DESCRIPTION
**Pull request by estrattonbailey**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copy this to `.env` and `.env.test` files
+
 SENTRY_AUTH_TOKEN=
+EXPO_PUBLIC_ENV=development
 EXPO_PUBLIC_LOG_LEVEL=debug
 EXPO_PUBLIC_LOG_DEBUG=
```
Comment (estrattonbailey) [inline .env.example:6]: So one env change here is: Expo doesn't actually use `NODE_ENV`, instead it uses that value to load a `.env.*` file. E.g. `NODE_ENV=test` loads `.env.test`.

File: src/view/com/feeds/FeedSourceCard.tsx [hunk 2]
```diff
@@ -169,7 +169,9 @@ export function FeedSourceCardLoaded({
 
         {showSaveBtn && feed.type === 'feed' && (
           <View>
+            {console.log(`feed-${feed.displayName}-toggleSave`)}
```
Comment (pfrazee) [inline src/view/com/feeds/FeedSourceCard.tsx:172]: Leftover log

File: index.js [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
 import {registerRootComponent} from 'expo'
 import {doPolyfill} from '#/lib/api/api-polyfill'
-doPolyfill()
 
 import App from '#/App'
 
+doPolyfill()
+
+if (IS_TEST) {
+  LogBox.ignoreAllLogs() // suppress all logs in tests
```
Comment (estrattonbailey) [inline index.js:14]: Error alerts were covering up UI and failing tests.

File: src/view/com/util/Toast.tsx [hunk 4]
```diff
@@ -7,13 +7,15 @@ import {colors} from 'lib/styles'
...
 import {useAnimatedValue} from 'lib/hooks/useAnimatedValue'
+import {IS_TEST} from '#/env'
 
 const TIMEOUT = 4e3
 
 export function show(
   message: string,
   _icon: FontAwesomeProps['icon'] = 'check',
 ) {
+  if (IS_TEST) return
```
Comment (pfrazee) [inline src/view/com/util/Toast.tsx:18]: Holy shit. Yes. Thank you. Oh my god.

**Review by pfrazee — APPROVED** [review]
Good stuff man, just one leftover log

File: jest/test-pds.ts [hunk 5]
```diff
@@ -59,17 +59,21 @@ export async function createServer(
...
-      publicUrl: pdsUrl,
-      inviteRequired,
+      hostname: 'localhost',
       dbPostgresSchema: `pds_${id}`,
+      inviteRequired,
     },
     bsky: {
       dbPostgresSchema: `bsky_${id}`,
+      port: port3,
+      publicUrl: 'http://localhost:2584',
```
Comment (estrattonbailey) [inline jest/test-pds.ts:76]: Updates for latest dev-env package

File: src/view/com/modals/UserAddRemoveLists.tsx [hunk 6]
```diff
@@ -182,7 +187,7 @@ function ListItem({
           <ActivityIndicator />
         ) : (
           <Button
-            testID={`user-${subject}-addBtn`}
+            testID={`user-${handle}-addBtn`}
```
Comment (estrattonbailey) [inline src/view/com/modals/UserAddRemoveLists.tsx:190]: Didn't know how to get a match here aside from passing in the handle, seems innocuous enough

File: src/view/com/util/Toast.tsx (hunk 4 above)
Comment (estrattonbailey) [inline src/view/com/util/Toast.tsx:18]: Toasts also conceal UI and fail tests. We should really just test Toasts separately.

File: src/view/shell/desktop/Search.tsx [hunk 7]
```diff
@@ -42,6 +42,7 @@ export function SearchResultCard({
 
   return (
     <Link
+      testID={`searchAutoCompleteResult-${profile.handle}`}
```
Comment (estrattonbailey) [inline src/view/shell/desktop/Search.tsx:45]: Dropped this in the refactor, whoops

**Pull request merged**